app/models/nonprofit.rb

```diff
@@ -538,4 +541,14 @@ def has_any_social?
   def timezone_is_valid
     timezone.blank? || ActiveSupport::TimeZone.all.map { |t| t.tzinfo.name }.include?(timezone) || errors.add(:timezone, "is not a valid timezone")
   end
+
+  def enforce_two_factor_requirement
+    return unless require_two_factor?
+
+    users.where(otp_required_for_login: false).find_each do |user|
```

Comment:
At current scale doing this synchronously may not be hugely problematic but still might be better to be async. Is there a good way to do that already set up?